src/components/AuthorsCards/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import {
+	Wrapper,
+	AuthorsTitle,
+	Card1,
+	Card2,
+	Card3,
+	Card4,
+	CardContent,
+	CardText,
+	CardTitle,
+	CardSubtitle,
+	IconBar,
+	CardIcon,
+} from './styled'
+import { LinkedinIn, Github, Instagram } from '@styled-icons/fa-brands'
+
+const AuthorsCards: React.FC = () => (
+	<Wrapper>
+		<AuthorsTitle>PROGRAMÁTOŘI (ČTI: KOUZELNÍCI)</AuthorsTitle>
+
+		{/* Jindřich Máca */}
+		<Card1>
+			<CardContent>
+				<CardText>
+					<CardTitle>Jindřich Máca</CardTitle>
+					<CardSubtitle>
+						~ full-stack, functional programming addict ~
+					</CardSubtitle>
+				</CardText>
+				<IconBar>
+					<CardIcon
+						href="https://www.linkedin.com/in/macajind/"
+						target="_blank"
+					>
+						<LinkedinIn />
+					</CardIcon>
+					<CardIcon
+						href="https://github.com/macajind"
+						target="_blank"
+					>
+						<Github />
+					</CardIcon>
+				</IconBar>
+			</CardContent>
+		</Card1>
```

Comment:
I think it would be better if you did one component for one author and then changed the information based on the props. There would be less code and it would be clearer 🙂

Reply:
Done! This was harder then expected, but Jindra helped me. :3 Truly saved us a lot of space! 
![savedspace](https://user-images.githubusercontent.com/76131194/108770865-24837200-755b-11eb-8492-647c2f8168ec.png)

Reply:
Great 👍